addons/ui/RscDisplayOptionsLayout.hpp

```diff
@@ -12,7 +12,7 @@ class RscDisplayOptionsLayout {
             onMouseButtonUp = "with uiNamespace do {['mouseButtonUp',_this,''] call RscDisplayOptionsLayout_script;};";
         };
         #define ADD_ELEMENT(var1) class Element##var1: Element022 {\
-            idc = __EVAL(12000 + var1);\
+            idc = 12##var1;\
```

Comment:
```suggestion
            idc = QUOTE(12000 + (var1));\
```

Reply:
right now it's a number which is ideal
but `1+1 = 11` type "addition" is weird and hard to understand
so I'm fine making it a statement that needs to be evaluated

Reply:
It's a plus b. What's not to understand? This addresses the issue while keeping the original numbers.

Reply:
(12##023) = 12023 = (12000 + 023)